notebooks/lesson10_clusters.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<ul><li>"feature vector" -&gt; "feature matrix" for consistency in previous lessons</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>